input/intro-notes/StructureDefinition-au-patient-intro.md

```diff
@@ -15,5 +15,38 @@
   - If `Patient.communication.preferred` has not been included, or is set to "false", systems should understand this as the language for the interpreter service is not known.
 - See each Identifier profile page for guidance related to that identifier type.
 
+**Person Recorded Sex Or Gender implementation guidance**
+  This profile supports patient sex and gender information currently in use in various official documents and services using the [Person Recorded Sex Or Gender](https://hl7.org/fhir/extensions/5.1.0-snapshot1/StructureDefinition-individual-recordedSexOrGender.html) extension, as per the guidance in the following table.
+  <table border="1">
+    <thead>
+    <tr>
+    <th>Sex and gender information to represent</th>
+    <th>extension.value</th>
+    <th>extension:type</th>
+    <th>extension:sourceDocument</th>
+    <th>extension:sourceField</th>
+    <th>extension:jurisdiction</th>
+    </tr>
+    </thead>
+    <tbody>
+    <tr>
+    <td>Sex and gender information in use in documents, e.g. Australian passport or driver's license</td>
+    <td> </td>
+    <td>'document'</td>
+    <td>Use an appropriate value from <a href="ValueSet-common-au-recorded-sex-or-gender-source-document-type.html">Common AU Recorded Sex or Gender (RSG) Source   Document Type</a> value set, e.g. 'passport', '53245-7 Driver license' </td>
+    <td>name of the field where this information is   held in the document, e.g. 'Sex'</td>
+    <td>Use an appropriate value from <a href="ValueSet-common-au-recorded-sex-or-gender-source-document-jurisdiction.html">Common AU Recorded Sex or Gender (RSG) Source   Document Jurisdiction</a> value set, e.g. 'AU' if representing sex and gender information in an Australia passport</td>
+    </tr>
+    <tr>
+    <td>Sex and gender information in use in   services and infrastructure, e.g. HI Services or My Health Record</td>
+    <td></td>
+    <td>Use an appropriate value from <a href="ValueSet-common-au-recorded-sex-or-gender-type.html">Common AU Recorded Sex or Gender Type</a> value set, e.g. 'au-hi-service', 'au-my-health-record'</td>
```

Comment:
Clarify the conformance requirement. "SHALL use the appropriate value from ... if any of the codes within the value set can apply to the concept being communicated."